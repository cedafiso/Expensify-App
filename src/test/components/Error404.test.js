import React from 'react';
import {shallow} from 'enzyme';
import Error404 from '../../componentes/Error404';

test('It should render the not found page correctly', () => {
    const wrapper = shallow(<Error404 />);
    expect(wrapper).toMatchSnapshot();
});