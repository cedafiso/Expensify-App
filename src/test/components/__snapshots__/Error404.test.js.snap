// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render the not found page correctly 1`] = `
<div>
  ERROR 404-
  <Link
    to="/"
  >
    Go home
  </Link>
</div>
`;
